lib/collection/url-match-pattern-list.js

```diff
@@ -0,0 +1,88 @@
+var _ = require('../util').lodash,
```

Comment:
Ideally, any class that is defined inside `lib/collection` should represent a data structure that exists inside the Postman Collection Format. 

This is mostly a helper class, so it should not be inside `lib/collection`. Can you move this to a new folder, `lib/url-pattern/match-list.js`?